broadinstitute/pearl
# AR-76 study dashboard

This ports over the study environment dashboard from the previous prototype.  No UX improvements are made, only technical polishing (e.g. ditching the css files in favor of inline style).

This also adds populate scripts

TO TEST:
1. nuke/reset your database
2. start the admin API app
3. run `./scripts/populate-setup.sh`
4. run `./scripts/populate-portal.sh ourhealth`
5. start the admin UI
6. go to localhost:3000 and login
7. confirm you can click "configure" on the "Longitudinal Heart Health Study" and see something like below (none of the links will work yet...) 
![image](https://user-images.githubusercontent.com/2800795/208180245-7f979280-0a87-46cb-ab88-97bfee5177ec.png)
